lib/models/ARN.js

```diff
@@ -0,0 +1,94 @@
+const errors = require('../errors');
+
+const validServices = {
+    aws: ['s3', 'iam'],
```

Comment:
We now have `sso`, `ring`, `utapi`, `sts` as valid services for ARN.
https://github.com/scality/Arsenal/blob/master/lib/policyEvaluator/RequestContext.js#L133
May be we should include them as well.

Reply:
updated